cli/mender-artifact/partition.go

```diff
@@ -57,20 +57,24 @@ type vFile int
 
 // Open is a utility function that parses an input image and path
 // and returns a V(irtual)P(artition)File.
-func (v vFile) Open(comp artifact.Compressor, imgpath, key string) (VPFile, error) {
+func (v vFile) Open(comp artifact.Compressor, imgpath string) (VPFile, error) {
 	imgname, fpath, err := parseImgPath(imgpath)
 	if err != nil {
 		return nil, err
 	}
-	modcands, isArtifact, err := getCandidatesForModify(imgname, []byte(key))
+	modcands, isArtifact, err := getCandidatesForModify(imgname)
 	if err != nil {
 		return nil, err
 	}
+	if modcands == nil {
```

Comment:
Same as the others. Why isn't no partitions returned an error?

Reply:
See previous comments.